modules/http/http.h

```diff
@@ -87,6 +88,8 @@ void http_dd_set_cipher_suite(LogDriver *d, const gchar *ciphers);
 gboolean http_dd_set_tls13_cipher_suite(LogDriver *d, const gchar *tls13_ciphers);
 void http_dd_set_proxy(LogDriver *d, const gchar *proxy);
 gboolean http_dd_set_ssl_version(LogDriver *d, const gchar *value);
+void http_dd_set_accept_encoding(LogDriver *d, const gchar *accept);
```

Comment:
[nitpick]

`const gchar *encoding` might explain the value a bit better here.

Reply:
[antinitpick]

You are right.

Reply:
Done